frontend/src/context/layers/admin_level_data.ts

```diff
@@ -233,6 +233,7 @@ export const fetchAdminLevelDataLayerData: LazyLoader<AdminLevelDataLayerProps>
       const data: { [key: string]: any }[] = (
         await (
           await fetch(datedPath, {
+            method: 'POST',
```

Comment:
I think this will break other layers. The post method is already added with the logic above:

```
      const options = requestBody
        ? {
            method: 'POST',
            headers: { 'Content-Type': 'application/json' },
            body: JSON.stringify(requestBody),
          }
        : {};
```

Reply:
yep